app/src/main/java/com/jerboa/db/AppDB.kt

```diff
@@ -353,6 +366,24 @@ val MIGRATION_12_13 = object : Migration(12, 13) {
     }
 }
 
+val MIGRATION_13_14 = object : Migration(13, 14) {
```

Comment:
I think if these two features are grouped together than can be a single migration.